aeon/dj_pipeline/report.py

```diff
@@ -10,12 +10,18 @@
 import pandas as pd
 
 from aeon.analysis import plotting as analysis_plotting
+from aeon.dj_pipeline.analysis.visit import Visit, VisitEnd
+from aeon.dj_pipeline.analysis.visit_analysis import *
 
 from . import acquisition, analysis, get_schema_name
 
 schema = dj.schema(get_schema_name("report"))
 os.environ["DJ_SUPPORT_FILEPATH_MANAGEMENT"] = "TRUE"
 
+experiment_name = "exp0.2-r0"
+MIN_VISIT_DURATION = 24  # in hours (minimum duration of visit for analysis)
+WHEEL_DIST_CRIT = 1  # in cm (minimum wheel distance travelled)
+MIN_BOUT_DURATION = 1  # in seconds (minimum foraging bout duration)
```

Comment:
Let's not make these global variables in this module, you can just define them wherever they're actually used.

Reply:
these functions are called within make methods. You want these values to be hard-coded as an argument?

Reply:
Yes, either hardcode or make them variable scoped within the `make()` only, should not be global variables.